www/include/common/common-Func.php

```diff
@@ -2091,3 +2091,49 @@ function get_child($id_page, $lcaTStr)
     $redirect = $DBRESULT->fetchRow();
     return $redirect;
 }
+
+/**
+ * Get the select option.
+ *
+ * @return array<int, int>
+ */
+function getSelectOption()
+{
+    $stringToArray = function ($value) {
+        if (strpos($value, ',') !== false) {
+            return explode(',', $value);
+        }
+        return array($value);
```

Comment:
need to confirm with @kduret but due to recent changes we can now declare array() as []  on 2.8.x :smiley:

Reply:
I keep the same usage throughout the file